src/libraries/System.Private.CoreLib/src/System/Activator.RuntimeType.cs

```diff
@@ -100,6 +100,8 @@ public static partial class Activator
 
         [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2006:UnrecognizedReflectionPattern",
             Justification = "Implementation detail of Activator that linker intrinsically recognizes")]
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:UnrecognizedReflectionPattern",
```

Comment:
Why are `IL2006` and `IL2026` both `UnrecognizedReflectionPattern` ? Shouldn't different warning numbers have different titles/names?

Reply:
@mateoatr to comment on the second parameter values...

Reply:
Anything that is put after the colon in the CheckId parameter is used only for documentation purposes (this is, the warning suppression mechanism completely ignores what the user puts after the colon, it only cares for the four numbers of the warning id). 
We sometimes put a subcategory on the warnings to further categorize them, although [this specific warning](https://github.com/mono/linker/blob/master/src/linker/Linker.Dataflow/ReflectionMethodBodyScanner.cs#L1250) does not have the `UnrecognizedReflectionPattern` subcategory, I don't know if we should add it.

Reply:
It is super confusing as a reader to see 2 warning numbers with the same string. What is the title of `IL2026`? `RequiresUnreferencedCode`? We should update it here so this doesn't look like duplicated attributes.

Reply:
Renamed to RequiresUnreferencedCode.